x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/plugin/RestEqlSearchAction.java

```diff
@@ -32,10 +32,13 @@
 
 import static org.elasticsearch.rest.RestRequest.Method.GET;
 import static org.elasticsearch.rest.RestRequest.Method.POST;
+import static org.elasticsearch.xpack.ql.util.LoggingUtils.logQueryFailure;
+import static org.elasticsearch.xpack.ql.util.LoggingUtils.wrapWithFailureLogging;
 
 @ServerlessScope(Scope.PUBLIC)
 public class RestEqlSearchAction extends BaseRestHandler {
-    private static Logger logger = LogManager.getLogger(RestEqlSearchAction.class);
+    private static final Logger LOGGER = LogManager.getLogger(RestEqlSearchAction.class);
```

Comment:
👍 on making sure the errors are reported as coming from the SearchAction and NOT from a wrapping listener.